app/src/test/java/com/hedvig/app/util/coroutines/MainCoroutineRule.kt

```diff
@@ -0,0 +1,26 @@
+package com.hedvig.app.util.coroutines
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.TestCoroutineScope
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+@ExperimentalCoroutinesApi
+class MainCoroutineRule(
+    private val dispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
```

Comment:
Should this `dispatcher` be exposed, in order to allow access to `runBlockingTest`? 🤔